word_game.java

```diff
@@ -22,8 +23,12 @@ public static void main (String [] args ) {
 
         System.out.println ("what is ur age");
         int age = scanner.nextInt();
-
-        System.out.println ("there once was a person named" + name + "who lived in " + city + ".At the age of " + age + "," + name + "went to collage at "+ collage + "." + name + " graduated and went to work as " + proffesional + ". Then, " + name + "adopted a(n) Animal named" + petname + ". They both lived happily ever after!" );
+                            
+        /*
+            Adding printf to display an output.
+        */
+        System.out.printf("There once was a person named %s who lived in %s. At the age of %d, %s went to college at %s. %s graduated and went to work as %s. Then %s adopted a(n) %s named %s. They both lived happily ever after!", name, city, age, name, collage, name, professional, name, animal_type, petname);
```

Comment:
Seems like you have 2 types of professional, one for professional and another one for proffesional